### PR TITLE
issue #34 Delete Gmail service.

### LINKED ISSUE #34: Remove GMail based mail service
Remove GMail mail service and all related data as it's not needed anymore

Sabtasks:
- [x] #116 Find all connection to GMail service
- [x] #118 Remove GMail services and all related date

### PR DESCRIPTION
**Pull request by Myroslavv**
### Summary

Delete GmailAccountMailService.cs.

closes #34 

### Areas Of Interest
None

### Checklist
- #116 Find all connection to GMail service.
- #118 Remove GMail services and all related date.
#### Design
None

#### Perfromance
None

#### Unit Tests
None

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs [hunk 1]
```diff
@@ -24,11 +24,7 @@ public void RegisterDependencies(IIocContainer container)
             {
                 container.Register<IMailService, DebugMailService>(IocLifetimeEnum.Scoped);
             }
-            else if (Is<IISDeployment>.Enabled)
-            {
-                container.Register<IMailService, GmailAccountMailService>(IocLifetimeEnum.Scoped);
-            }
-            else
+            else if (Is<IISDeployment>.Disabled)
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: Now we might have potential problem. If either IISDeployment and DebugMode flags are not set properly we won't have IMailService registered and app will fail.

Make sure DebugMailService is registered if nothing else is registered.

Comment (Myroslavv) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: default

Comment (Myroslavv) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: Do you mean this?

Comment (sdiachen) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: yes

**Pull request merged**